src/USER-MISC/pair_lebedeva.cpp

```diff
@@ -0,0 +1,437 @@
+/* ----------------------------------------------------------------------
+   LAMMPS - Large-scale Atomic/Molecular Massively Parallel Simulator
+   http://lammps.sandia.gov, Sandia National Laboratories
+   Steve Plimpton, sjplimp@sandia.gov
+
+   Copyright (2003) Sandia Corporation.  Under the terms of Contract
+   DE-AC04-94AL85000 with Sandia Corporation, the U.S. Government retains
+   certain rights in this software.  This software is distributed under
+   the GNU General Public License.
+
+   See the README file in the top-level LAMMPS directory.
+------------------------------------------------------------------------- */
+
+/* ----------------------------------------------------------------------
+   Contributing author: Zbigniew Koziol 
+   (National Center for Nuclear Research, Poland)
+   e-mail: softquake at gmail dot com
+   Writing this was based on C code of Kolmogorov-Crespi potential 
+   of Jaap Kroes and others.
+  
+   This is potential described in
+   [Lebedeva et al., Physica E, 44(6), 949-954, 2012.]
+------------------------------------------------------------------------- */
+
+#include <math.h>
+#include <stdio.h>
+#include <stdlib.h>
+#include <string.h>
+#include "pair_lebedeva.h"
+#include "atom.h"
+#include "comm.h"
+#include "force.h"
+#include "neigh_list.h"
+#include "memory.h"
+#include "error.h"
+
+using namespace LAMMPS_NS;
+
+#define MAXLINE 1024
+#define DELTA 4
+
+/* ---------------------------------------------------------------------- */
+
+PairLebedeva::PairLebedeva(LAMMPS *lmp) : Pair(lmp)
+{
+  writedata = 1;

```

Comment:
This is copied from the (incorrect) template. This style does not write coefficients to the data file and thus this line should be deleted. Since it also does not provide a `Pair::single()` method, it should have the `single_enable = 0;` statement, that is missing here (default is to assume single exists).